packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -87,6 +92,7 @@ tests:
           csharp: BsonJavaScript
           shell: Code
           ruby: BSON::Code
+          rust: ''
```

Comment:
can we use `Bson::JavaScriptCode` here?

Reply:
This is testing "partial" constructions, i.e. transpilations of fragmentary input.  In this case, it's testing translating the code "symbol" without any arguments.  Because for Rust we have to switch between `JavaScriptCode` and `JavaScriptCodeWithScope` based on whether the scope argument is present, the symbol itself has to be empty.  I don't think this matters much because it's just testing nonsense.